pkg/services/provisioning/dashboards/config_reader_test.go

```diff
@@ -20,6 +23,14 @@ func TestDashboardsAsConfig(t *testing.T) {
 	t.Run("Dashboards as configuration", func(t *testing.T) {
 		logger := log.New("test-logger")
 
+		sqlstore.InitTestDB(t)
+		for _, index := range []int{1, 2} {
+			orgCommand := models.CreateOrgCommand{Name: fmt.Sprintf("Main Org. %v", index)}
+			if err := sqlstore.CreateOrg(&orgCommand); err != nil {
+				t.Fatal(err)
+			}
```

Comment:
We use `require.NoError` to verify an error didn't occur.
```suggestion
			err := sqlstore.CreateOrg(&orgCommand)
			require.NoError(t, err)
```